cgi/product_jqm_multilingual.pl

```diff
@@ -138,8 +138,32 @@
 		}
 	}
 	my @param_langs = keys %param_langs;
+	
+	# 01/06/2019 --> Yuka always sends fr fields even for Spanish products, try to correct it
+
+	if ((defined $User_id) and ($User_id eq 'kiliweb') and (defined param('cc')) and (param('cc') ne "fr")) {
+	
+		my $param_cc = param('cc');
+		
+		my %lc_overrides = (
```

Comment:
You could put the whole `lc_override` logic into some `sub` in Products.pm to avoid code duplication in here. 🙂